model-services/main.py

```diff
@@ -27,8 +28,21 @@
 best_pallet_config = None
 val_list = None
 
+origins = [
+    "http://localhost",
+    "http://localhost:8080",
+]

```

Comment:
can you add this url here:

_https://dsd.commanderkowalski.uk_

this is deployed backend